.circleci/config.yml

```diff
@@ -93,11 +101,26 @@ jobs:
             echo 'export ORG_GRADLE_PROJECT_mavenCentralPassword="$SONATYPE_NEXUS_TOKEN_PASSWORD"' >> "$BASH_ENV"
             echo 'export ORG_GRADLE_PROJECT_signingInMemoryKey="$SIGNING_GPG_IN_MEMORY"' >> "$BASH_ENV"
             echo 'export ORG_GRADLE_PROJECT_signingInMemoryKeyPassword="$GPG_SIGNING_KEY_PW_NEW"' >> "$BASH_ENV"
-      - run:
-          name: Publish snapshot to Maven Central
-          command: ./gradlew publishAllPublicationsToMavenCentralRepository
+      # TODO Remove this condition to allow releasing non-SNAPSHOT versions. 
+      - when:
+          condition: << parameters.snapshot >>
+          steps:
+            - run:
+                name: Publish to Maven Central
+                command: ./gradlew publishAllPublicationsToMavenCentralRepository
       - android/save-gradle-cache
       - android/save-build-cache
+  create-github-release:
```

Comment:
Nitpick but I would leave an empty line between jobs. I think that's how we've been doing it?

Reply:
Ah good catch, done!